src/app/content/components/Page/PageComponent.tsx

```diff
@@ -130,9 +139,17 @@ export default class PageComponent extends Component<PagePropTypes> {
 
   public render() {
     const pageIsReady = this.props.page && this.props.textSize !== null;
+    const PT = this.props.inAssigned ?
```

Comment:
this fix is for a different card, and i think we need to work on how to communicate the nav height a bit more, can you move it to a different pr?